third_party/2and3/xxhash.pyi

```diff
@@ -0,0 +1,29 @@
+from _typeshed import ReadableBuffer
+from hashlib import _Hash
+
+VERSION: str
+XXHASH_VERSION: str
+
+class _IntDigestHash(_Hash):
+    def intdigest(self) -> int: ...
+
+# python-xxhash v2.0.0 does not support the string or usedforsecurity kwargs
+def xxh32(__string: ReadableBuffer = ...) -> _IntDigestHash: ...
```

Comment:
Technically these are classes, not functions returning an `_IntDigestHash`.

Reply:
They're structured like this for parallelism with the hashlib stubs, which return `_Hash`.

Reply:
The hashlib stubs use this structure: should that be retained?

Reply:
That doesn't have to be done in this PR, but in general it's better to mirror the runtime objects as closely as possible.